examples/rig-kafka-pub-sub-example/docker-starter/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  zk:
+    image: confluentinc/cp-zookeeper:5.0.0
+    container_name: zk
+    ports:
+      - 2181:2181
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+    restart: on-failure:5
+
+  kafka:
+    image: confluentinc/cp-kafka:5.0.0
+    container_name: kafka
+    ports:
+      - 9092:9092 # container communication
+      - 9094:9094 # localhost communication
+    environment:
+      KAFKA_ZOOKEEPER_CONNECT: zk:2181
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://kafka:9092,OUTSIDE://localhost:9094
+      KAFKA_LISTENERS: INSIDE://0.0.0.0:9092,OUTSIDE://0.0.0.0:9094
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT
+      KAFKA_INTER_BROKER_LISTENER_NAME: INSIDE
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "true"
+      KAFKA_NUM_PARTITIONS: 4
+    restart: on-failure:5
+
+  reactive-interaction-gateway:
+    container_name: reactive-interaction-gateway
+    image: accenture/reactive-interaction-gateway:2.1.1
```

Comment:
Let's remove tag `2.1.1` and use latest version.

Reply:
just a thought, but wouldn't it make sense to build the image from `../../../..`? This way it would be consistent with the rest of the code in the repository, regardless of what's currently published on Docker Hub..

Reply:
I will make the update according to @mmacai 's comment. I like Kevin's idea, but I'm not a Docker wiz and would need a little support to make sure I'm implementing doing that correctly.

Reply:
I ran into this issue once I updated to latest. Any ideas?

```
~/projects/reactive-interaction-gateway/examples/rig-kafka-pub-sub-example/docker-starter (pub-sub-example)⚡ ./start.sh
Removing network docker-starter_default
Creating network "docker-starter_default" with the default driver
Pulling example (example:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling example (example:latest)...
ERROR: pull access denied for example, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Reply:
For example like this https://github.com/Accenture/reactive-interaction-gateway/blob/master/examples/tests/run_all_tests.sh#L14, switch to root, build image, switch back, run compose

Reply:
I did not implement this. It currently just pulls the image from Docker Hub.

Reply:
Would you mind if I take a look and potentially commit the change?